packages/bierzo-wallet/src/routes/transactions/components/sorting.ts

```diff
@@ -6,12 +6,12 @@ export interface SortingStateProps {
   readonly order: SortOrder;
 }
 
-export const ORDER_ASC = 1;
-export const ORDER_DESC = -1;
 export type SortOrder = 1 | -1;
+export const ORDER_ASC: SortOrder = 1;
+export const ORDER_DESC: SortOrder = -1;
 
-export const TX_AMOUNT_COLUMN = "Amount";
-export const TX_DATE_COLUMN = "Date";
+export const TX_AMOUNT_COLUMN = "Amount" as const;
```

Comment:
What is this for (`as const`)?

Reply:
When you assign a literal to a variable, TypeScript does type widening, i.e. the variable has a wider type than the literal. E.g.

```ts
const a = "abc"; // type of a is string, not "abc"
const b = 123; // type of b is number, not 123
const c = [1, 2, 3]; // type of c is number[], not readonly [1, 2 3]
```

with the `as const` language construct (new in TS3.4) you can stop this widening, e.g

```ts
const a = "abc" as const; // type of a is "abc"
const b = 123 as const; // type of b is 123
const c = [1, 2, 3] as const; // type of c is readonly [1, 2 3]
```

In this case, the type of `TX_DATE_COLUMN` changed from `string` to `"Date"`. Now `TX_DATE_COLUMN` can be assigned to a variable of type `TxsOrder`, which does not accept general `string`s. `TX_AMOUNT_COLUMN` changes to type `"Amount"`, which can be assigned to `readonly name: "Amount" | "Transactions";`.

Reply:
thank you